components/centraldashboard/public/components/dashboard-view.js

```diff
@@ -1,6 +1,7 @@
 import '@polymer/iron-icon/iron-icon.js';
 import '@polymer/iron-icons/iron-icons.js';
 import '@polymer/paper-card/paper-card.js';
+import '@polymer/paper-ripple/paper-ripple.js';
```

Comment:
Can we make sure `@polymer/paper-ripple` is included as a dependency in package.json? It's obviously being included by one of our other dependencies but since we are importing it specifically here, we should declare it as such.

Reply:
That's a good point!